CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 ------------------------------------------------------------------------
 - Feature: [#2697] Add initial OpenGraphics custom assets.
 - Change: [#2708] "Show Options Window" keyboard shortcut now works in the title screen.
+- Change: [#2758] Screenshots are now saved in a "screenshots" folder.
```

Comment:
We should probably clarify that the folder is in the config / custom content directory.

Reply:
I agree, but I wasn't sure how to word that and would appreciate help with the changelog entry

Reply:
Something like this?
```suggestion
- Change: [#2758] Screenshots are now saved in a dedicated `screenshots` subfolder in OpenLoco's user config folder.
```